src/pytest_dbfixtures/plugin.py

```diff
@@ -61,6 +61,14 @@ def pytest_addoption(parser):
         dest='logsdir',
     )
 
+    parser.addoption(
+        '--dynamodb-dir',
```

Comment:
Hmm, I think `dir` is better in this case (we want path to dir, not path to file). The word "path" mostly means "path to file/dir". You can't say "dir to file". Ok, you can say; you can say anything, but you know what I mean ;p